RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -0,0 +1,184 @@
+            }
+
+            Spacer()
+
+            ManageSubscriptionsButtonsView(viewModel: viewModel,
+                                           openURL: openURL)
+        }
+        .onAppear {
+            checkAndLoadSubscriptionInformation()
+        }
+    }
+
+}
+
+@available(iOS 15.0, *)
+private extension ManageSubscriptionsView {
+
+    func checkAndLoadSubscriptionInformation() {
+        if !viewModel.isLoaded {
+            Task {
+                await viewModel.loadSubscriptionInformation()
+            }
+        }
+    }
+
+}
+
+@available(iOS 15.0, *)
+struct HeaderView: View {
+    var body: some View {
+        Text("How can we help?")
+            .font(.title)
+            .padding()
+    }
+}
+
+@available(iOS 15.0, *)
+struct SubscriptionDetailsView: View {
+    let subscriptionInformation: SubscriptionInformation
+    let refundRequestStatus: String?
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            Text("\(subscriptionInformation.title) - \(subscriptionInformation.duration)")
+                .font(.subheadline)
+                .padding(.horizontal)
+                .padding(.top)
+
+            Text("\(subscriptionInformation.price)")
+                .font(.caption)
+                .foregroundColor(Color.gray)
+                .padding(.horizontal)
+
+            Text("\(subscriptionInformation.renewalString): \(subscriptionInformation.nextRenewal)")
+                .font(.caption)
+                .foregroundColor(Color.gray)
+                .padding(.horizontal)
+                .padding(.bottom)
+
+            if let refundRequestStatus = refundRequestStatus {
+                Text("Refund request status: \(refundRequestStatus)")
+                    .font(.caption)
+                    .bold()
+                    .foregroundColor(Color.gray)
+                    .padding(.horizontal)
+                    .padding(.bottom)
+            }
+        }
+    }
+}
+
+@available(iOS 15.0, *)
+struct ManageSubscriptionsButtonsView: View {
+
+    @ObservedObject
+    var viewModel: ManageSubscriptionsViewModel
+    let openURL: OpenURLAction
+    @State
+    private var showRestoreAlert: Bool = false
+
+    var body: some View {
+        VStack(spacing: 16) {
+            if let configuration = viewModel.configuration {
+                ForEach(configuration.paths, id: \.id) { path in
+                    Button(path.title.en_US) {
+                        handleAction(for: path)
+                    }
+                    .restorePurchasesAlert(isPresented: self.$showRestoreAlert)
+                    .buttonStyle(ManageSubscriptionsButtonStyle())
+                }
+            }
+
+            Button("Contact support") {
+                Task {
+                    openURL(URLUtilities.createMailURL()!)
+                }
+            }
+            .padding()
+        }
+    }
+
+    private func handleAction(for path: CustomerCenterData.HelpPath) {
```

Comment:
Should this method be in the view model? It looks like it to me, other than `showRestoreAlert` would need moving in there, unsure on that.